src/pages/contact.js

```diff
@@ -20,8 +21,21 @@ const ContactPage = () => {
       <SEO title="Contact" />
       <h1>Contact Info</h1>
       <p>{contactInfo.contact.email}</p>
-      <p><a href={contactInfo.contact.links.twitter}>twitter</a></p>
-      <p><a href={contactInfo.contact.links.facebook}>facebook</a></p>
+      <p>
+        <a href={contactInfo.contact.links.twitter} target="_blank">
+          Twitter
+        </a>
+      </p>
+      <p>
+        <a href={contactInfo.contact.links.facebook} target="_blank">
+          Facebook
+        </a>
+      </p>
+      <p>
+        <a href={contactInfo.contact.links.meetup} target="_blank">
+          Meetup
```

Comment:
@BayLadyCoder Can you add `rel="noreferrer` to the links? There's a warning that comes up when running it:
<img width="578" alt="Screen Shot 2020-11-01 at 8 35 23 AM" src="https://user-images.githubusercontent.com/21016245/97804365-5de74900-1c1d-11eb-8c86-cf9afa3e0640.png">

```suggestion
        <a href={contactInfo.contact.links.twitter} target="_blank" rel="noreferrer">
          Twitter
        </a>
      </p>
      <p>
        <a href={contactInfo.contact.links.facebook} target="_blank" rel="noreferrer">
          Facebook
        </a>
      </p>
      <p>
        <a href={contactInfo.contact.links.meetup} target="_blank" rel="noreferrer">
          Meetup
```

Reply:
@alexxmitchell Working on it. Thanks for pointing it out! :)

Reply:
@alexxmitchell Please review. I just pushed the changes (Added rel="noreferrer" to all the links).